TACC/Core-Portal
# FP-387: UI Fix: Truncate

## Overview: ##

Truncate via CSS until JavaScript solution is available.

See [request on PR for FP-387](https://github.com/TACC/Frontera-Portal/pull/29#pullrequestreview-420205889).

## PR Status: ##

* Work in Progress.
* Hold.

## Related Jira tickets: ##

* [FP-387](https://jira.tacc.utexas.edu/browse/FP-387)

## Summary of Changes: ##

- Use new hack class.
- ~~Import and document global styles.~~ (removed this scope creep)
- Create and document `styles/` directory.
- Create new hack classes `_truncate-approx-char-count(…)`.
- Create new utility classes `u-ellipsis` (used by hack class).

## Testing Steps: ##
1. Load Dashboard.
2. Ensure "Job ID" (a.k.a. Job Name) and "Output Location" values are truncated.

## Notes: ##
